packages/hooks/src/hooks/useWallet.ts

```diff
@@ -22,16 +22,56 @@ export function useWallet() {
     connected,
     provider,
     correctNetwork,
+    network,
   } = context;
 
   React.useEffect(() => {
     if (userAddress && provider && chainId === NETWORKS.mainnet) {
-      provider.lookupAddress(userAddress).then((address) => {
+      provider.lookupAddress(userAddress).then(address => {
         setEns(address as string);
       });
     }
   }, [userAddress, provider]);
 
+  const switchToCorrectNetwork = async () => {
+    if (window.ethereum) {
+      try {
+        console.log('chainId', { chainId });
+        console.log('requiredNetwork', { network });
+        // check if the chain to connect to is installed
+        await window.ethereum.request({
+          method: 'wallet_switchEthereumChain',
+          params: [{ chainId: `0x${network}` }], // chainId must be in hexadecimal numbers
+        });
+      } catch (error) {
+        console.error(error);
+        // This error code indicates that the chain has not been added to MetaMask.
+        // If it is not, then install it into the user MetaMask
+        // @ts-ignore
```

Comment:
Instead of `@ts-ignore`, did you try using something `@ts-expect-error`? I think that would be more appropriate.

Reply:
No, I haven't tried it, I'll try it.